spec/stack.js

```diff
@@ -39,6 +39,16 @@ describe('Stack', () => {
         .to.alter(() => stack.length(), { from: 2, to: 1 })
     })
 
+    it('takes an element from the top of the stack.', () => {
+      const stack = new Stack()
+      stack.push('bob')
+      stack.push('alice')
+      stack.push('carl')
+
+      expect(stack.pop())
+        .to.equal('carl')
+    })
```

Comment:
would you want to test that the element is removed from the array value of stack?